.travis.yml

```diff
@@ -31,6 +31,8 @@ matrix:
           env: deps=high
         - php: 7.1
           env: deps=low
+        - php: 7.2
+          env: deps=high
```

Comment:
no need to add a new section, 7.1 should be changed to  7.2 just above instead, deps=low